### ISSUE TITLE
Error on downloading files

### ISSUE THREAD

**Issue by mlopezgva** [Docheinstein/music-dragon#1]
Hi

Each time I try to download a file, I get roughly the same error:

```
[youtube] Extracting URL: https://music.youtube.com/watch?v=euYswki5Gak                                                                                                                                                              ⇣91.53 KiB/s ⇡7.48 KiB/s 192.168.1.24   86.127.229.124   99%   866.7 Mbps 
[youtube] euYswki5Gak: Downloading webpage
[youtube] euYswki5Gak: Downloading ios player API JSON
[youtube] euYswki5Gak: Downloading web creator player API JSON
[youtube] euYswki5Gak: Downloading ios music player API JSON
[youtube] euYswki5Gak: Downloading web music client config
[youtube] euYswki5Gak: Downloading player 6db2bd17
[youtube] euYswki5Gak: Downloading web music player API JSON
[youtube] euYswki5Gak: Downloading m3u8 information
[youtube] euYswki5Gak: Downloading m3u8 information
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/music_dragon/repository.py", line 1278, in fetch_track_info_callback
    fetch_youtube_track_streams_callback(track_id, ytt.streams)
  File "/usr/local/lib/python3.8/dist-packages/music_dragon/ui/mainwindow.py", line 1664, in track_streams_fetched
    audios = sorted([s for s in yttrack.streams if s["type"] == "audio"], key=lambda s: s["size"])
TypeError: '<' not supported between instances of 'NoneType' and 'NoneType'
WARN: download attempt n. 0 failed for video euYswki5Gak: ValueError could not convert string to float: '\x1b[0;94m  0.0%\x1b[0m'
WARN: download attempt n. 1 failed for video euYswki5Gak: ValueError could not convert string to float: '\x1b[0;94m  0.1%\x1b[0m'
ERROR: all download attempts (2 fails for video euYswki5Gak: could not convert string to float: '\x1b[0;94m  0.1%\x1b[0m'
WARN: no track with video id = euYswki5Gak was in download
```

I've tried with 20-30 different songs of different (3) albums.
I have just installed the software, so it should be up-to-date.

**Comment by mlopezgva** [comment]
UPDATE:

This happens (to  me) when running the program from the cli (`$ music-dragon`); when launched with the gui (like <kbd>F2</kbd> music-dragon <kbd>ENTER</kbd>) it works as expected.

**Comment by Docheinstein** [comment]
Thank you for reporting the problem.

It should have been fixed with 80709c175eb2d98761ff3eeaf8de40f17a4f28f5.

Let me know if you still have problems.

**Comment by mlopezgva** [comment]
Working!

That was fast. :-)
Thank you!
